Tasks/AppCenterDistributeV1/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -27,8 +27,8 @@
   "loc.input.help.releaseNotesInput": "Anmerkungen zu dieser Version.",
   "loc.input.label.releaseNotesFile": "Datei mit Anmerkungen zu dieser Version",
   "loc.input.help.releaseNotesFile": "Wählen Sie eine UTF-8-codierte Textdatei aus, die die Anmerkungen zu dieser Version enthält.",
-  "loc.input.label.destinationId": "Ziel-ID",
-  "loc.input.help.destinationId": "Die ID der Verteilergruppe oder des Speichers, in der bzw. dem die App bereitgestellt wird. Lassen Sie den Eintrag leer, um die Standardgruppe zu verwenden.",
+  "loc.input.label.destinationIds": "Ziel-ID",
```

Comment:
FYI the resource files for other languages don't need to be updated. They are updated later by the loc process.